### PR TITLE
Updated an access modifier of an object decleration in Sandbox.java

### PR DESCRIPTION
**Pull request by sandeshShahapur**
### Overview
I spotted a todo comment that specified to not have the access specifier of the object `classHandler` (declaration) in line 23 as `public`. Therefore I resolved it by changing the access modifier to `private`. I hope this is valid (I'm new), thank you!

### Proposed Changes
```diff
- public ClassHandler classHandler; // todo not public
+ private ClassHandler classHandler;
```

### CODE REVIEW

File: sandbox/src/main/java/org/robolectric/internal/bytecode/Sandbox.java [hunk 1]
```diff
@@ -20,7 +20,7 @@ public class Sandbox {
   private final SandboxClassLoader sandboxClassLoader;
   private final ExecutorService executorService;
   private ShadowInvalidator shadowInvalidator;
-  public ClassHandler classHandler; // todo not public
+  private ClassHandler classHandler; // todo not public
```
Comment (utzcoz) [inline sandbox/src/main/java/org/robolectric/internal/bytecode/Sandbox.java:23]: Remove this todo comment too?

**Review by utzcoz — CHANGES_REQUESTED** [review]

**Comment by utzcoz** [comment]
@sandeshShahapur Please squash your commits to one.

**Comment by sandeshShahapur** [comment]
> @sandeshShahapur Please squash your commits to one.

@utzcoz  can you please guide me on how to do that?

**Comment by utzcoz** [comment]
> > @sandeshShahapur Please squash your commits to one.
> 
> @utzcoz can you please guide me on how to do that?

There are many tutorials on the internet: e.g. https://www.git-tower.com/learn/git/faq/git-squash/.

**Comment by utzcoz** [comment]
@sandeshShahapur You need to rebase on the latest master with only one commit.

**Comment by utzcoz** [comment]
Hi @sandeshShahapur , after long time no-response, I will close this PR. Feel free to reopen it if you want to continue this PR. Thanks.

**Pull request closed without merging**